styles/prosilver/template/show_tag.html

```diff
@@ -47,9 +47,11 @@ <h2 class="forum-title">{RH_TOPICTAGS_SEARCH_HEADER}</h2>
 		<!-- EVENT viewforum_body_topic_row_before -->
 		<li class="row<!-- IF topicrow.S_ROW_COUNT is even --> bg1<!-- ELSE --> bg2<!-- ENDIF --><!-- IF topicrow.S_POST_GLOBAL --> global-announce<!-- ENDIF --><!-- IF topicrow.S_POST_ANNOUNCE --> announce<!-- ENDIF --><!-- IF topicrow.S_POST_STICKY --> sticky<!-- ENDIF --><!-- IF topicrow.S_TOPIC_REPORTED --> reported<!-- ENDIF -->">
 			<!-- EVENT viewforum_body_topic_row_prepend -->
-			<dl class="row-item {topicrow.TOPIC_IMG_STYLE}">
```

Comment:
What was the source of your comparison? As on the official phpbb 3.2.x branch [1], the `dl` tag has the `class="row-item"`
[1] https://github.com/phpbb/phpbb/blob/3.2.x/phpBB/styles/prosilver/template/viewforum_body.html#L158